document/payment-receipt.go

```diff
@@ -0,0 +1,117 @@
+package document
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/go-pdf/fpdf"
+)
+
+type CustomerInfo struct {
+	Fullname   string
+	Address    string
+	PostalCode string
+	City       string
+	Province   string
+	Email      string
+	Phone      string
+}
+
+type TransactionInfo struct {
+	PolicyCode     string
+	EffectiveDate  string
+	ExpirationDate string
+	PriceGross     string
+	NextPayment    string
+	PayDate        string
+}
+
+type ReceiptInfo struct {
+	CustomerInfo CustomerInfo
+	Transaction  TransactionInfo
+}
+
+func PaymentReceipt(info ReceiptInfo) ([]byte, error) {
+	var (
+		err error
+		buf bytes.Buffer
+	)
+	pdf := initFpdf()
+
+	woptaHeader(pdf, false)
+
+	woptaFooter(pdf)
+
+	pdf.AddPage()
+
+	pdf.SetX(130)
+
+	text := "Egr./Gent.le/Spett.le\n" +
+		info.CustomerInfo.Fullname + "\n" +
+		info.CustomerInfo.Address + "\n" +
+		info.CustomerInfo.PostalCode + " " + info.CustomerInfo.City + " (" + info.CustomerInfo.Province + ")\n" +
+		info.CustomerInfo.Email + " - " + info.CustomerInfo.Phone
+
+	setBlackBoldFont(pdf, standardTextSize)
+	pdf.MultiCell(0, 4, text, "", fpdf.AlignLeft, false)
+
+	pdf.Ln(15)
+
+	text = "Oggetto: Quietanza di pagamento polizza n. " + info.Transaction.PolicyCode
+
+	setBlackRegularFont(pdf, standardTextSize)
+	pdf.MultiCell(0, 4, text, "", fpdf.AlignLeft, false)
+
+	pdf.Ln(5)
+
+	text = "Gentile Cliente,"
+	pdf.MultiCell(0, 4, text, "", fpdf.AlignLeft, false)
+
+	pdf.Ln(5)
+
+	text = "la presente ricevuta è da considerarsi valida come quietanza di pagamento, Le consigliamo pertanto di " +
+		"conservarla con la documentazione del Suo contratto assicurativo."
+	pdf.MultiCell(0, 4, text, "", fpdf.AlignLeft, false)
+
+	pdf.Ln(5)
+
+	text = "La ringraziamo e Le porgiamo i nostri più cordiali saluti."
+	pdf.MultiCell(0, 4, text, "", fpdf.AlignLeft, false)
+
+	pdf.Ln(17)
+
+	text = "RICEVUTO DI PAGAMENTO DEL PREMIO"
```

Comment:
Haven't seen the requirements and could be an error on my part, but shouldn't it be RICEVUT**A** ?